compete.html

```diff
@@ -130,12 +130,12 @@ <h3>Building a robot</h3>
       <a name="signup"></a><h2>Want to compete?</h2>
       <div class="column m-8-12 m-offset-2-12 text-left">
         <p>
-          Please register your interest below, we'll let you know when formal entries open later this year.
+          Please register your interest below. We'll let you know once places are confirmed. We expect to confirm places at the end of September 2019.
         </p>
         <p class="text-heavy">
           You must be over 18 to run a Student Robotics team and so you must also be over 18 to fill in this form.
         </p>
-        {% include team-signup.html %}
```

Comment:
It's occurred to me that by removing this we're also losing the GDPR statement:
> Student Robotics will use the information you provide on this form to contact you around
 leading a team at Student Robotics. We will contact you via email.
> We use MailChimp as our contacting platform. By clicking below to sign up, you acknowledge that your information will be transferred to MailChimp for processing. Learn more about MailChimp's privacy practices <a href="https://mailchimp.com/legal/" target="_blank">here</a>.

We should probably reinstate this or something very similar.

Reply:
We've solved this by adding it directly to the form.